apps/governance/src/routes/staking/home/validator-tables/standby-pending-validators-table.tsx

```diff
@@ -153,14 +162,12 @@ export const StandbyPendingValidatorsTable = ({
               calculatesPerformancePenalty(performanceScore),
               2
             ),
-            [ValidatorFields.OVERSTAKING_PENALTY]: formatNumberPercentage(
-              calculateOverstakedPenalty(id, allNodesInPreviousEpoch),
-              2
-            ),
-            [ValidatorFields.TOTAL_PENALTIES]: formatNumberPercentage(
-              calculateOverallPenalty(id, allNodesInPreviousEpoch),
-              2
-            ),
+            [ValidatorFields.OVERSTAKING_PENALTY]: overstakingPenalty
+              ? formatNumberPercentage(overstakingPenalty, 2)
+              : '-',
+            [ValidatorFields.TOTAL_PENALTIES]: totalPenalty
```

Comment:
**nitpick**: all of this could be handled inside `formatNumberPercentage`

Reply:
Making this change was the right move, but revealed some failing tests in develop! I blame you!